apps/i18n/dance/en_us.json

```diff
@@ -10,8 +10,25 @@
   "danceAiFeedbackKeyPressRightOrLeft": "⚠️ Make sure to press at least one of the arrow keys while your dance is running.",
   "danceAIFeedbackMatchDancer": "⚠️ You must have the same dancer selected in the make new dancer block and the dancer do new move block.",
   "danceAiFeedbackNeedNewDancer": "⚠️ You need to make a dancer.",
-  "danceAiFeedbackNewMove": "⚠️ You need to add a block to make the dancer do a new move.",  
-  "danceAiFeedbackNoAiBlock": "⚠️ Add the `A.I. generate effect` block to setup.",  
+  "danceAiFeedbackNewMove": "⚠️ You need to add a block to make the dancer do a new move.",
+  "danceAiFeedbackNoAiBlock": "⚠️ Add the `A.I. generate effect` block to setup.",
+  "danceAiModalHeading": "generate {input} effect",
+  "danceAiModalChooseEmoji": "Choose three emojis.",
+  "danceAiModalGenerateButton": "Generate",
+  "danceAiModalFinding": "A.I. is finding the best effect to generate.",
+  "danceAiModalGenerating": "A.I. is generating this effect.",
+  "danceAiModalEffectButton": "Effect",
+  "danceAiModalEffect": "A.I. generated this effect.",
+  "danceAiModalUseEffectButton": "Use effect",
+  "danceAiModalCodeButton": "Code",
+  "danceAiModalCode": "Did you know? A.I. wrote this code to generate the effect.",
+  "danceAiModalUseCodeButton": "Use code",
+  "danceAiModalStartOverButton": "Start over",
+  "danceAiModalExplanationButton": "Learn more",
+  "danceAiModalRegenerateButton": "Regenerate",
+  "danceAiModalExplanation": "Each emoji contributed to A.I.'s decision. Here are five possible effects and the calculations that A.I. made.",
```

Comment:
Just curious. This text is displayed underneath the explanation bar graphs. The sentence  'Here are five possible and the calculations that A.I. made' doesn't seem to match the bar graphs. The bar graphs are showing the calculations for all the possible output effects.

Reply:
Good observation.  There is a revised design for the explanation mode for which this string will be appropriate.